### DISCUSSION TITLE
NVM Migration Question

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js#7182] [General]
Maybe a dumb question, and if so feel free to tell me to RTFM....

Should one be able to NVM migrate between two 800 series controllers, with the 1st being on a NEWER SDK version than the 2nd one?

Example:
800 series controller running **7.22.1** --> Different brand 800 series controller running **7.21.3**

?

Thanks,
Jason

**Comment by AlCalzone** [comment]
With Z-Wave JS, yes. Just **don't** check the "skip compatibility check" box.

**Reply by Botched1** [reply]
Interestingly if I DO NOT check the skip compatibility check, I get this message:
![image](https://github.com/user-attachments/assets/3a9ce84e-89b4-411c-aa04-61fd0404fd75)

```
2024-09-21 09:10:26.917 INFO Z-WAVE: jsonToNVM cannot convert to a pre-7.0 NVM version. Use jsonToNVM500 instead. (ZW0322) restoreNVM undefined
2024-09-21 09:10:46.907 INFO APP: GET /health/zwave 301 0.883 ms - 191
```

But they are both zwave 800 controllers...

**Reply by AlCalzone** [reply]
Please share both an NVM backup of the source and the target controller. I need to look at them to figure out why the target is seemingly not supported.

**Reply by Botched1** [reply]
What is the best way to send them to you? Wasn't sure if posting it here was a great idea or not.

Source = Zooz ZST39
![image](https://github.com/user-attachments/assets/c2afa2c7-0b0b-4af8-89eb-458d01cc08d0)


Destination = Mixtile 2in1 ( https://www.mixtile.com/store/accessory/mixtile-2-in-1-zigbee-z-wave-mpcie-interface-module/ )
![image](https://github.com/user-attachments/assets/052b1359-7417-4c2f-ba9a-4f623545b365)

**Reply by Botched1** [reply]
@AlCalzone Here are the two NVM backups.
[NVM Files.zip](https://github.com/user-attachments/files/17090829/NVM.Files.zip)

**Comment by Botched1** [comment]
Bit of an update... Mixtile released a 7.22.1 based firmware this morning. I'll update to that, try the NVM backup/restore again and see if I get any different results.

**Comment by AlCalzone** [comment]
How old is your version of Z-Wave JS? I cannot reproduce with the `nvmedit` CLI, which should be doing the exact same steps, only that it reads the NVM backups from file.

**Reply by Botched1** [reply]
I jus tried it again this time with both source and destination on the same firmware rev - 7.22.1. Same error.

Currently running:
zwave-js-ui: 9.20.0
zwave-js: 13.3.1


Doing the NVMe restore from the zwave js ui.

**Reply by Botched1** [reply]
<s>It's really not a big deal. Just another random Chinese controller I was going to test out. Not my production system or anything. 

I already killed 1 board with a bad flash, so am down to 1 left (assuming I haven't killed it yet, need to test).</s>

EDIT: Nope, I killed them both. So I have nothing left to test with on this model. Feel free to put this aside, as there is no point chasing it if I can't even confirm anything.

Sorry to waste your time!
